File: agent/bindings/kotlin/lib/src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/RemoteRepo_ListParams.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class RemoteRepo_ListParams(
  val query: String? = null,
  val first: Long,
  val afterId: String? = null,
)

File: agent/bindings/csharp/RemoteRepoListParams.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class RemoteRepoListParams
  {
    [JsonProperty(PropertyName = "query")]
    public string Query { get; set; }
    [JsonProperty(PropertyName = "first")]
    public int First { get; set; }
    [JsonProperty(PropertyName = "afterId")]
    public string AfterId { get; set; }
  }
}
